### PR TITLE
Remove obsolete URLs and references to Ripple

### PR DESCRIPTION
**Pull request by cryptobrad**
## High Level Overview of Change

Remove the old comments about Hebbian learning and out dated information and broken links to old Ripple wiki. Its sad that this is needed since the name was taken by the Ripple company but we will clean it up
 

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [X] Documentation Updates
- [ ] Release

### CODE REVIEW

**Review by nbougalis — APPROVED** [review]
Looks good. Some of the markdown files were outdated or not particularly helpful and it's ok that they're deleted but they could _probably_ have been salvaged.

In a perfect world, almost every folder would contain a markdown document with at least a small description.

File: src/ripple/protocol/impl/PublicKey.cpp [hunk 1]
```diff
@@ -106,7 +106,7 @@ sliceToHex(Slice const& slice)
     @param strict_param whether to enforce strictly canonical semantics
 
     @note For more details please see:
-    https://ripple.com/wiki/Transaction_Malleability
+    https://xrpl.org/transaction-malleability.html#transaction-malleability
```
Comment (mDuo13) [inline src/ripple/protocol/impl/PublicKey.cpp:109]: This link will be a little more stable if you don't include the anchor.

```suggestion
    https://xrpl.org/transaction-malleability.html
```

File: src/ripple/app/tx/impl/Escrow.cpp [hunk 2]
```diff
@@ -58,22 +58,22 @@ namespace ripple {
     expiration time, the XRP can only be returned to the sender.
 
     For more details on escrow, including examples, diagrams and more please
-    visit https://ripple.com/build/escrow/#escrow
+    visit https://xrpl.org/escrow.html#escrow
```
Comment (mDuo13) [inline src/ripple/app/tx/impl/Escrow.cpp:61]: This link will be more stable without the anchor.

```suggestion
    visit https://xrpl.org/escrow.html
```

File: src/ripple/app/paths/README.md [hunk 3]
```diff
@@ -1,137 +0,0 @@
-Theory of Pathfinding
```
Comment (mDuo13) [inline src/ripple/app/paths/README.md:1]: This document still has some kind-of useful information in it? You're right that it's extremely dated though.

File: src/ripple/app/paths/Pathfinder.cpp [hunk 4]
```diff
@@ -444,8 +444,7 @@ Pathfinder::computePathRanks(int maxPaths)
 static bool
 isDefaultPath(STPath const& path)
 {
-    // TODO(tom): default paths can consist of more than just an account:
-    // https://forum.ripple.com/viewtopic.php?f=2&t=8206&start=10#p57713
+    // FIXME: default paths can consist of more than just an account:
```
Comment (mDuo13) [inline src/ripple/app/paths/Pathfinder.cpp:447]: For future reference, xrpl.org has some discussion of [default paths](https://xrpl.org/paths.html#default-paths) which may be useful for fixing this up.

No action needed here at this time though.

**Review by mDuo13 — APPROVED** [review]
Thank you for doing this. I left a couple suggestions (mostly to make the links more likely to continue to work if any changes happen on the docs side) but I think you can consider all of those optional nice-to-haves.

File: src/ripple/app/paths/README.md (hunk 3 above)
Comment (nbougalis) [reply src/ripple/app/paths/README.md:1]: I went through this and honestly, there's very little captured in that doc that is helpful. I think it's better off removed for now.

**Comment by nbougalis** [comment]
@cryptobrad, I think @mDuo13's URL tweaks help, so I just used GitHub's web-based UI to commit suggestions to the PR. They should be squashed down when this commit is merged.

**Pull request closed without merging**